utils/config/daos_server.yml

```diff
@@ -249,15 +249,14 @@
 #
 #  scm_mount: /mnt/daos/1
 #
-#  # Backend SCM device type. Either use DCPM (datacentre persistent memory)
+#  # Backend SCM device type. Either use DCPM (datacenter persistent memory)
```

Comment:
I do not know, but I'm wondering if 'datacenter' should be two words to match the acronym?  I found 'AppDirect' in the Intel literature online as two words.  Do we need / want to maintain that to match the Intel naming convention, or is it intentional to make it different?

Reply:
changing to "data-center" and "app-direct" respectively, thanks

Reply:
@cdurf1 does the above change makes sense? also should I remove references to DCPM and if so what should I replace it with?

Reply:
@cdurf1 
When referring to the persistent memory, do we need to refer to it as "Intel Optane Data Center Persistent Memory", or otherwise maintain Intel branding?  Or do we specifically not want to have any Intel branding on the words used to describe the DCPM?  Is 'data-center' and 'app-direct' approved usage, or should it be referred to differently?

Reply:
Intel(R) Optane(TM) <noun> is the proper format. The Names Database (http://prod-namesdb.intel.com/) offers the following nouns. We generically refer it to Intel(R) Optane(TM) memory technology. datacenter does not need/require a hyphen. DCPM is not an approved Intel use externally.

The approved usage is: Intel® Optane™

Approved nouns:
brand
logo
mark
media
memory
Memory and Storage Management
memory media
memory module
memory ready
memory technology
name
persistent memory
Persistent Memory Controller Exporter (PMCE)
Persistent Memory Debug Tool
Persistent Memory Power and Thermal Utility
PMem
Solid State Drive
SSD
storage
storage media
storage technology
technology
trademark

Reply:
I'm surprised NVMe is not on there? I'm not updating the configuration variable name for scm_class "dcpm"  as part of this PR, does this also need to be done @mjmac @johannlombardi  @cdurf1